documentation/examples/prometheus-kubernetes.yml

```diff
@@ -76,6 +76,12 @@ scrape_configs:
   relabel_configs:
   - action: labelmap
     regex: __meta_kubernetes_node_label_(.+)
+  - target_label: __address__
+    replacement: kubernetes.default.svc.cluster.local:443
```

Comment:
please leave out the `cluster.local` – it should work without due to the search domains that Kubernetes sets up.